singer-connectors/transform-field/requirements.txt

```diff
@@ -1 +1 @@
-pipelinewise-transform-field==2.0.0
+-e git+https://github.com/transferwise/pipelinewise-transform-field.git@AP-928#egg=pipelinewise-transform-field
```

Comment:
https://github.com/transferwise/pipelinewise-transform-field/pull/21 ready to merge :yay:

Reply:
Done